server/views/pages/view/view.njk

```diff
@@ -16,40 +16,30 @@
         </div>
     </div>
 
-    <form method="POST">
-        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-        <input type="hidden" name="licenceId" value="{{ licence.id }}">
-
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds-from-desktop">
-
-                {{ inductionMeetingDetails(licence) }}
-
-                {{ additionalConditions(licence) }}
-
-                {{ bespokeConditions(licence) }}
-
-                <p class="govuk-body">
-                    {{ govukButton({
-                        text: "Return to case list",
-                        classes: "govuk-button--secondary govuk-!-margin-bottom-0 govuk-body",
-                        href: '/licence/view/cases',
-                        attributes: { 'data-qa': 'return-to-view-list' }
-                    }) }}
-                </p>
-            </div>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+            {{ inductionMeetingDetails(licence) }}
+            {{ additionalConditions(licence) }}
+            {{ bespokeConditions(licence) }}
 
-            <div class="govuk-grid-column-one-third-from-desktop">
+            <p class="govuk-body">
                 {{ govukButton({
-                    text: "Print licence",
-                    classes: "govuk-button--primary",
-                    type: "submit",
-                    name: "result",
-                    value: "print",
-                    attributes: { 'data-qa': 'print-licence' }
+                   text: "Return to case list",
+                   classes: "govuk-button--secondary govuk-!-margin-bottom-0 govuk-body",
+                   href: '/licence/view/cases',
+                   attributes: { 'data-qa': 'return-to-view-list' }
                 }) }}
-            </div>
+            </p>
+        </div>
 
+        <div class="govuk-grid-column-one-third-from-desktop">
+            {{ govukButton({
```

Comment:
Why not put this into the same `div`? For spacing just?

Reply:
It keeps the print button at the top on normal screen sizes - two-thirds column for the licence view, one third for the print button, but moves to the bottom on smaller screens.